### DISCUSSION TITLE
How to backup my board?

### DISCUSSION THREAD

**Discussion by don-ferris** [kanboard/kanboard#5500] [Q&A]
Is there a way to backup my entire Kanboard instance - all projects, tasks, users, everything?

**Comment by fguillot** [comment]
Backing up your Kanboard instance typically involves copying the `data` folder and creating a SQL dump of your database depending how you have configured your instance.

Refer to the official documentation for your chosen database (PostgreSQL, MySQL, or SQLite) regarding the SQL dump as the process may vary across different database systems.
